include/zephyr/bluetooth/gap.h

```diff
@@ -78,6 +78,309 @@ extern "C" {
 #define BT_LE_AD_GENERAL                0x02 /* General Discoverable */
 #define BT_LE_AD_NO_BREDR               0x04 /* BR/EDR not supported */
 
+/* Appearance Values Last Modified on 2023-01-05 */
+#define BT_APPEARANCE_UNKNOWN                          0x0000 /* Generic Unknown */
```

Comment:
Not something we've done a lot of yet, but we could consider moving these to a `enum bt_appearance` and use that new enum type in `bt_get_appearance` and `bt_set_appearance`. However those will require a (non-breaking) stable API change, so maybe best to leave for another PR. 

@hermabe Thoughts?

Reply:
Yeah, I think an enum is generally better, but that can be done in a different PR.